frontend/src/components/SyncEditor/decorate.ts

```diff
@@ -101,18 +125,29 @@ const addChangeDecorations = (
     decorations: any[]
 ) => {
     changes.forEach((chng) => {
-        const { $t, $a } = chng
+        const { $t, $a, $f } = chng
         const obj: any = get(change.mappings, $a)
         if (obj) {
+            //if (!isCustomEdit) {
```

Comment:
@jeswanke Should probably remove this line?

Reply:
ah, yeah--cleanup :)